src/api/index.ts

```diff
@@ -0,0 +1,108 @@
+import express from 'express';
+import { body, validationResult } from 'express-validator';
+import { PrismaClient } from '@prisma/client';
+import logger from '../logger';
+
+const prisma = new PrismaClient();
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  return res.json({ msg: 'The API is working' });
+});
+
+/**
+ * @description creates a subscription for a deviceToken adding the provided wallet address
+ * @param {string} deviceToken - the device token of user's device (e.g. phone)
+ * @param {string} walletAddress - the walletAddress of a user
```

Comment:
IMO, `address` has to be an object. It has to have e.g. a network name, which eventually also a Enum (need to be verified, throw error in case of not support)

And the address needs to be validated, however it's fine to address it later with another issue.

Reply:
Good point, we need to validate the address. 👍🏼 

We don't need to store the network because the service will run for a specific network already and by validating the address we will be checking that it is supported